client/client.go

```diff
@@ -104,11 +104,10 @@ func (c *Client) CreateVirtualChannel(objectiveRequest virtualfund.ObjectiveRequ
 }
 
 // CloseVirtualChannel attempts to close and defund the given virtually funded channel.
-func (c *Client) CloseVirtualChannel(channelId types.Destination, paidToBob *big.Int) protocols.ObjectiveId {
+func (c *Client) CloseVirtualChannel(channelId types.Destination) protocols.ObjectiveId {
```

Comment:
Now the client just relies on vouchers that have been exchanged to figure out the correct amounts.